rough-ideas.md

```diff
@@ -5,14 +5,15 @@ These are a rough set of fields in the `.bimg` format.
 A line is a tuple of (text, text colors, background colors) in blit format.  
 Frames are lists of lines.  
 Image files may be a single frame, or list of frames.  
+Each frame can have a `palette` key to specify a specific palette for that frame, following the rules of the base `palette` key.
```

Comment:
This is for *image data*, not metadata. This should be sitting down in the metadata section prepended with `(per frame)`